README.md

```diff
@@ -132,7 +136,9 @@ far, `Method_1` and `Method_2` returned `'Empty` bodies, but we can fix that.
 import           Data.Text (Text)
 import qualified Serv.ContentType as Ct
 
-type Method_3 = 'Method 'GET '[] ('Body '[ Ct.TextPlain ] Text)
+type RawBody = HasBody '[ Ct.TextPlain ] Text
```

Comment:
Hmm, maybe. I intend to give it a rough pass then re-read for clarity. Maybe that'll give some insight into what sounds best?